code/src/App.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react'
+import React, { useState } from "react";
+
+import Survey from "./components/survey/Survey";
+import Summary from "./components/Summary";
 
 export const App = () => {
+  const [showSummary, setSummary] = useState(false);
```

Comment:
far better would be `[showSummary, setShowSummary]` or `[summary, setSummary]`

Reply:
Solved